src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -387,6 +391,45 @@
                DisplayName="When the output is updated" />
   </EnumProperty>
 
+  <DynamicEnumProperty Name="Trimming"
+                DisplayName="Trimming"
+                Description="Select the desired .NET trimming option for optimizing your application's deployment size and performance."
+                HelpUrl="https://learn.microsoft.com/dotnet/core/deploying/native-aot/?tabs=net7"
```

Comment:
We'll need a fwlink for any `HelpUrl`s in this PR. It might also be worth dropping the `?tabs=net7` suffix, so that it redirects to the latest version of .NET when the user clicks it. Ideally we'd forward to a version relative to the project, but that's out of scope for this PR.

Reply:
Oh, sorry, you called this out in your PR description.